countdownTimer.js

```diff
@@ -1,39 +1,80 @@
-// Set the date we're counting down to
-var countDownDate = new Date("Mar 15, 2022 15:37:25").getTime();
+let timer = "5:00"
```

Comment:
Hmmm, a string works for now, but compared to using numbers, would make it harder if you needed to implement features like "add 20 seconds".

To make things even easier, what do you think you could do with a method like this: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now